core_types/src/chain_header.rs

```diff
@@ -24,6 +25,9 @@ pub struct ChainHeader {
     entry_type: EntryType,
     /// Key to the entry of this header
     entry_address: Address,
+    /// Address(es) of the agent(s) that authored and signed this entry.
+    /// Backed by the entry_signature below.
+    sources: Vec<Address>,
     /// agent's cryptographic signature of the entry
```

Comment:
So, just checking, the reason to put the source in the header itself is to not have to store it as meta-data when published to the DHT, and, in the future use cases of multiple signers of an entry, because in the local chain where there is no meta-data, you would have to do something else (like put the address of the signers in the entry itself).  But otherwise, we could be getting the address from the meta data (if on the dht), or by looking for the most recent agent entry on the chain.  Am I missing something?